crates/sqlbuiltins/src/functions/scalars.rs

```diff
@@ -0,0 +1,870 @@
+    example => "cos(0)",
+    description => "Compute the cosine of a number."
+);
+make_const!(
+    COT,
+    name => "cot",
+    example => "cot(1)",
+    description => "Compute the cotangent of a number."
+);
+make_const!(
+    COSH,
+    name => "cosh",
+    example => "cosh(0)",
+    description => "Compute the hyperbolic cosine of a number."
+);
+make_const!(
+    DEGREES,
+    name => "degrees",
+    example => "degrees(1)",
+    description => "Converts an angle measured in radians to an approximately equivalent angle measured in degrees."
+);
+make_const!(
+    EXP,
+    name => "exp",
+    example => "exp(1)",
+    description => "Compute the base-e exponential of a number."
+);
+make_const!(
+    FACTORIAL,
+    name => "factorial",
+    example => "factorial(5)",
+    description => "Compute the factorial of a number."
+);
+make_const!(
+    FLOOR,
+    name => "floor",
+    example => "floor(1.1)",
+    description => "Compute the largest integer less than or equal to a number."
+);
+make_const!(
+    GCD,
+    name => "gcd",
+    example => "gcd(10, 15)",
+    description => "Compute the greatest common divisor of two integers."
+);
+make_const!(
+    ISNAN,
+    name => "isnan",
+    example => "isnan(1)",
+    description => "Returns true if the argument is NaN, false otherwise."
+);
+make_const!(
+    ISZERO,
+    name => "iszero",
+    example => "iszero(0)",
+    description => "Returns true if the argument is zero, false otherwise."
+);
+make_const!(
+    LCM,
+    name => "lcm",
+    example => "lcm(10, 15)",
+    description => "Compute the least common multiple of two integers."
+);
+make_const!(
+    LN,
+    name => "ln",
+    example => "ln(1)",
+    description => "Compute the natural logarithm of a number."
+);
+make_const!(
+    LOG,
+    name => "log",
+    example => "log(1)",
+    description => "Compute the logarithm of a number in a specified base."
+);
+make_const!(
+    LOG10,
+    name => "log10",
+    example => "log10(1)",
+    description => "Compute the base-10 logarithm of a number."
+);
+make_const!(
+    LOG2,
+    name => "log2",
+    example => "log2(1)",
+    description => "Compute the base-2 logarithm of a number."
+);
+make_const!(
+    NANVL,
+    name => "nanvl",
+    example => "nanvl(1, 2)",
+    description => "Returns the first argument if it is not NaN, otherwise returns the second argument."
+);
+make_const!(
+    PI,
+    name => "pi",
+    example => "pi()",
+    description => "Compute the constant π."
+);
+
+make_const!(
+    POWER,
+    name => "pow",
+    example => "pow(2, 3)",
+    description => "Compute the power of a number."
+);
+make_const!(
+    RADIANS,
+    name => "radians",
+    example => "radians(1)",
+    description => "Converts an angle measured in degrees to an approximately equivalent angle measured in radians."
+);
+make_const!(
+    RANDOM,
+    name => "random",
+    example => "random()",
+    description => "Compute a random number between 0 and 1."
```

Comment:
I think I'd use "generates" or "returns" a random number. Also maybe designate if/how it's psudorandom?